common/state_traits/07_africa_traits.txt

```diff
@@ -97,4 +98,16 @@ state_trait_atlas_mountains = {
 		state_construction_mult = -0.10 #-0.15 Temp
 		state_infrastructure_mult = -0.15
 	}
+}
+
+state_trait_kalahari_desert = {
```

Comment:
Wrong place to add this, should be in ste_traits. These files should be vanilla only.